cntp-windeploy/main.cpp

```diff
@@ -56,11 +61,20 @@ int main(int argc, char** argv) {
 
     output << "Creating System Library Database...\n";
     output.flush();
-    SystemLibraryDatabase *libraryDatabase = new SystemLibraryDatabase(supportLibraryPaths);
+    SystemLibraryDatabase* libraryDatabase = new SystemLibraryDatabase(supportLibraryPaths);
 
     output << "Making folder self contained\n";
     output.flush();
     deployFolder.makeSelfContained(libraryDatabase);
 
+    if (parser.isSet("appx-output")) {
+        QFile::remove(parser.value("appx-output"));
+
+        QProcess makeappxProcess;
+        makeappxProcess.setProcessChannelMode(QProcess::ForwardedChannels);
+        makeappxProcess.start("makeappx", {"pack", "/d", deployFolder.destinationDir().absolutePath(), "/p", parser.value("appx-output")});
```

Comment:
We need to be able to make multi-architecture bundles too. Bonus if we can split the architecture-neutral files out as well

Reply:
I think we'll need to have some discussions about the multi-architecture bundles once support for cross compiling arm is complete